assignment/impl/src/java/org/sakaiproject/assignment/impl/AssignmentEventObserver.java

```diff
@@ -75,7 +75,7 @@ public void update(Observable o, Object arg) {
                                 Optional<AssignmentSubmission> submission = Optional.empty();
                                 // Assignments stores the gradebook item name and not the id :(, so we need to look it up
                                 try {
-                                    org.sakaiproject.service.gradebook.shared.Assignment gradebookAssignment = gradebookService.getAssignmentByNameOrId(event.getContext(), itemId);
+                                    org.sakaiproject.service.gradebook.shared.Assignment gradebookAssignment = gradebookService.getAssignmentByNameOrId(event.getContext(), event.getContext(), itemId);
```

Comment:
I'm not sure if we should leave a method with two arguments and then the gradebook service call the same method with three arguments, all of this seems very repetitive to me.